cmd/flowserver/expression.go

```diff
@@ -1,50 +0,0 @@
-package main
```

Comment:
Because the flow server is now a struct.. feels like it's handler functions should all be in the same file

Reply:
Really? why? Seems breaking it out gives you smaller files and logical seperation.

Reply:
I'm usually all for breaking things out but in this case it seems easier to find the server struct and its functions in a single server.go file